src/backend/main/py.main/caliopen_main/user/objects/settings.py

```diff
@@ -32,5 +32,5 @@ class Settings(ObjectUser):
     }
 
     _model_class = ModelSettings
-    _pkey_name = 'device_id'
+    _pkey_name = None
```

Comment:
I don't understand why you are removing the _pkey_name attribute. «Settings» is an `ObjectUser` ; it belongs to an user and has `user_id` as its primary key.